{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"posix_spawnattr_setexceptionports_np"},{"kind":"text","text":"("},{"kind":"externalParam","text":"_"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"UnsafeMutablePointer","preciseIdentifier":"s:Sp"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/posix_spawnattr_t","preciseIdentifier":"c:@T@posix_spawnattr_t","text":"posix_spawnattr_t"},{"kind":"text","text":"?>!, "},{"kind":"externalParam","text":"_"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/exception_mask_t","preciseIdentifier":"c:@T@exception_mask_t","text":"exception_mask_t"},{"kind":"text","text":", "},{"kind":"externalParam","text":"_"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/mach_port_t","preciseIdentifier":"c:@T@mach_port_t","text":"mach_port_t"},{"kind":"text","text":", "},{"kind":"externalParam","text":"_"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/exception_behavior_t","preciseIdentifier":"c:@T@exception_behavior_t","text":"exception_behavior_t"},{"kind":"text","text":", "},{"kind":"externalParam","text":"_"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/thread_state_flavor_t","preciseIdentifier":"c:@T@thread_state_flavor_t","text":"thread_state_flavor_t"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","text":"Int32","preciseIdentifier":"s:s5Int32V"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/nio\/posix_spawnattr_setexceptionports_np(_:_:_:_:_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/org.swift.docc\/documentation\/NIO\/posix_spawnattr_setexceptionports_np(_:_:_:_:_:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"role":"symbol","title":"posix_spawnattr_setexceptionports_np(_:_:_:_:_:)","roleHeading":"Function","fragments":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"posix_spawnattr_setexceptionports_np"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"UnsafeMutablePointer","preciseIdentifier":"s:Sp"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","text":"posix_spawnattr_t","preciseIdentifier":"c:@T@posix_spawnattr_t"},{"kind":"text","text":"?>!, "},{"kind":"typeIdentifier","text":"exception_mask_t","preciseIdentifier":"c:@T@exception_mask_t"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"mach_port_t","preciseIdentifier":"c:@T@mach_port_t"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"exception_behavior_t","preciseIdentifier":"c:@T@exception_behavior_t"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"thread_state_flavor_t","preciseIdentifier":"c:@T@thread_state_flavor_t"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","text":"Int32","preciseIdentifier":"s:s5Int32V"}],"symbolKind":"func","externalID":"c:@F@posix_spawnattr_setexceptionports_np","modules":[{"name":"NIO"}],"platforms":[{"beta":false,"unavailable":false,"name":"macOS","introducedAt":"10.5","deprecated":false}]},"hierarchy":{"paths":[["doc:\/\/org.swift.docc\/documentation\/NIO"]]},"references":{"doc://org.swift.docc/documentation/NIO/thread_state_flavor_t":{"role":"symbol","title":"thread_state_flavor_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"thread_state_flavor_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/thread_state_flavor_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"thread_state_flavor_t"}],"url":"\/documentation\/nio\/thread_state_flavor_t"},"doc://org.swift.docc/documentation/NIO/mach_port_t":{"role":"symbol","title":"mach_port_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"mach_port_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/mach_port_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"mach_port_t"}],"url":"\/documentation\/nio\/mach_port_t"},"doc://org.swift.docc/documentation/NIO/posix_spawnattr_setexceptionports_np(_:_:_:_:_:)":{"role":"symbol","title":"posix_spawnattr_setexceptionports_np(_:_:_:_:_:)","fragments":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"posix_spawnattr_setexceptionports_np"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"UnsafeMutablePointer","preciseIdentifier":"s:Sp"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","text":"posix_spawnattr_t","preciseIdentifier":"c:@T@posix_spawnattr_t"},{"kind":"text","text":"?>!, "},{"kind":"typeIdentifier","text":"exception_mask_t","preciseIdentifier":"c:@T@exception_mask_t"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"mach_port_t","preciseIdentifier":"c:@T@mach_port_t"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"exception_behavior_t","preciseIdentifier":"c:@T@exception_behavior_t"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"thread_state_flavor_t","preciseIdentifier":"c:@T@thread_state_flavor_t"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","text":"Int32","preciseIdentifier":"s:s5Int32V"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/posix_spawnattr_setexceptionports_np(_:_:_:_:_:)","kind":"symbol","type":"topic","url":"\/documentation\/nio\/posix_spawnattr_setexceptionports_np(_:_:_:_:_:)"},"doc://org.swift.docc/documentation/NIO":{"role":"collection","title":"NIO","abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO","kind":"symbol","type":"topic","url":"\/documentation\/nio"},"doc://org.swift.docc/documentation/NIO/posix_spawnattr_t":{"role":"symbol","title":"posix_spawnattr_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"posix_spawnattr_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/posix_spawnattr_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"posix_spawnattr_t"}],"url":"\/documentation\/nio\/posix_spawnattr_t"},"doc://org.swift.docc/documentation/NIO/exception_behavior_t":{"role":"symbol","title":"exception_behavior_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"exception_behavior_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/exception_behavior_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"exception_behavior_t"}],"url":"\/documentation\/nio\/exception_behavior_t"},"doc://org.swift.docc/documentation/NIO/exception_mask_t":{"role":"symbol","title":"exception_mask_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"exception_mask_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/exception_mask_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"exception_mask_t"}],"url":"\/documentation\/nio\/exception_mask_t"}}}